CRM/Core/Error.php

```diff
@@ -962,7 +964,7 @@ public static function exceptionHandler($pearError) {
    *   $obj
    */
   public static function nullHandler($obj) {
-    CRM_Core_Error::debug_log_message("Ignoring exception thrown by nullHandler: {$obj->code}, {$obj->message}");
+    CRM_Core_Error::debug_log_message("Ignoring exception thrown by nullHandler: {$obj->code}, {$obj->message}", FALSE, '', PEAR_LOG_ERR);
```

Comment:
I've been inconclusively bike-shedding on whether this should be `PEAR_LOG_DEBUG` or `PEAR_LOG_ERR`, but I don't think it matters much, so I'm good unless anyone has a strong opinion on this.

Reply:
These definitely may be legitimate errors that administrators may need to take action to correct.

If there is truly debug-level info being sent to the nullhandler, then some logic to sort that out from the errors could be a nice further improvement.